prov/efa/src/efa_domain.c

```diff
@@ -94,6 +107,25 @@ static int efa_open_device_by_name(struct efa_domain *domain, const char *name)
 		}
 	}
 
+	/*
```

Comment:
Nit: After moving the pd allocation inside of it, `efa_open_device_by_name()` is now doing more than what it seems from the name.